scalding-db/src/main/scala/com/twitter/scalding/db/macros/impl/ColumnDefinitionProviderImpl.scala

```diff
@@ -240,8 +243,10 @@ object ColumnDefinitionProviderImpl {
         val (box: Option[Tree], primitiveGetter: Tree) = cf.fieldType match {
           case "VARCHAR" | "TEXT" =>
             (None, q"""$rsTerm.getString($fieldName)""")
-          case "BOOLEAN" | "TINYINT" =>
```

Comment:
the reason for this is probably that in mysql `boolean` is the synonym of `tinyint(1)`.

Reply:
Yes, but shouldn't this be the right way to handle a case class with a `Byte` field?

Reply:
@gerashegalov wanted to follow up with you on this: If the field in the case class is declared as `Boolean` and the column in the DB is declared as a `TINYINT` things should work as before. However, if the case class declares the same field as a `Byte` instead, I've changed the implementation to handle the column in the DB as a byte since a `TINYINT` column can hold values in the range -128 to +127 (signed).

Reply:
@navinvishy tbh I was just giving my 2c for what I think the background was for the common treatment of BOOLEAN and TINYINT. If I can make a suggestion, I'd recommend to leave this change to another PR . It does not align with the title and intent to add support for blobs.

Reply:
@gerashegalov agreed. I've updated the PR to include only support for blobs. I'll add the support for tinyint and timestamp fields in a separate PR.